File: python/length_of_last.py
class Solution:
    def lengthOfLastWord(s: str) -> int:
        return len(s.split()[-1]) if s.split() else 0
    
print(Solution.lengthOfLastWord("Hello World"))
print(Solution.lengthOfLastWord("   fly me   to   the moon  "))
print(Solution.lengthOfLastWord("luffy is still joyboy"))
print(Solution.lengthOfLastWord(""))
print(Solution.lengthOfLastWord("a"))

File: leet/length_of_last.rb
def length_of_last_word(s)
    fixed = s.split
    if fixed == []
        return 0
    else
        return fixed[-1].length
    end
end

puts length_of_last_word("Hello World")
puts length_of_last_word("   fly me   to   the moon  ")
puts length_of_last_word("luffy is still joyboy")
puts length_of_last_word("")
puts length_of_last_word("a")